src/contracts/Protocol/BountyQuestion.sol

```diff
@@ -59,4 +62,8 @@ contract BountyQuestion is ERC721, ERC721Enumerable, ERC721URIStorage, ERC721Bur
     function supportsInterface(bytes4 interfaceId) public view override(ERC721, ERC721Enumerable) returns (bool) {
         return super.supportsInterface(interfaceId);
     }
+
+    function getAuthor(address user) public view returns (uint256[] memory) {
```

Comment:
since it's public you don't actually need this!  `authors(address user)` will be auto-generated by solidity

Reply:
I can't find the correct format for the second argument regardless and I think it's just easier to read as well but curious to know what the format would be but not going down that rabbit hole at this time